[2022-08-18 16:32:37,147] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Satellite_Data.load_into_json manual__2022-08-18T16:26:53.777932+00:00 [queued]>
[2022-08-18 16:32:37,164] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Satellite_Data.load_into_json manual__2022-08-18T16:26:53.777932+00:00 [queued]>
[2022-08-18 16:32:37,164] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-18 16:32:37,164] {taskinstance.py:1377} INFO - Starting attempt 2 of 2
[2022-08-18 16:32:37,164] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-18 16:32:37,198] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): load_into_json> on 2022-08-18 16:26:53.777932+00:00
[2022-08-18 16:32:37,211] {standard_task_runner.py:52} INFO - Started process 1268 to run task
[2022-08-18 16:32:37,216] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Satellite_Data', 'load_into_json', 'manual__2022-08-18T16:26:53.777932+00:00', '--job-id', '69', '--raw', '--subdir', 'DAGS_FOLDER/data_loader.py', '--cfg-path', '/tmp/tmpbdg97dct', '--error-file', '/tmp/tmpk60ube1n']
[2022-08-18 16:32:37,217] {standard_task_runner.py:80} INFO - Job 69: Subtask load_into_json
[2022-08-18 16:32:37,323] {task_command.py:371} INFO - Running <TaskInstance: Satellite_Data.load_into_json manual__2022-08-18T16:26:53.777932+00:00 [running]> on host b4db7d4790c9
[2022-08-18 16:32:37,505] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Satellite_Data
AIRFLOW_CTX_TASK_ID=load_into_json
AIRFLOW_CTX_EXECUTION_DATE=2022-08-18T16:26:53.777932+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-18T16:26:53.777932+00:00
[2022-08-18 16:32:37,510] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-18 16:32:37,511] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'python3 /opt/***/dags/satelliteTrack.py']
[2022-08-18 16:32:37,543] {subprocess.py:85} INFO - Output:
[2022-08-18 16:32:40,073] {subprocess.py:92} INFO - <Response [500]>
[2022-08-18 16:32:40,075] {subprocess.py:92} INFO - Traceback (most recent call last):
[2022-08-18 16:32:40,075] {subprocess.py:92} INFO -   File "/opt/***/dags/satelliteTrack.py", line 108, in <module>
[2022-08-18 16:32:40,076] {subprocess.py:92} INFO -     load_satellite_data()
[2022-08-18 16:32:40,076] {subprocess.py:92} INFO -   File "/opt/***/dags/satelliteTrack.py", line 88, in load_satellite_data
[2022-08-18 16:32:40,076] {subprocess.py:92} INFO -     data = api_call()
[2022-08-18 16:32:40,076] {subprocess.py:92} INFO -   File "/opt/***/dags/satelliteTrack.py", line 70, in api_call
[2022-08-18 16:32:40,076] {subprocess.py:92} INFO -     raise MyError(resp, "GET fail on request for Starlink satellites")
[2022-08-18 16:32:40,076] {subprocess.py:92} INFO - __main__.MyError: (<Response [500]>, 'GET fail on request for Starlink satellites')
[2022-08-18 16:32:40,215] {subprocess.py:96} INFO - Command exited with return code 1
[2022-08-18 16:32:40,233] {taskinstance.py:1909} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 195, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2022-08-18 16:32:40,236] {taskinstance.py:1420} INFO - Marking task as FAILED. dag_id=Satellite_Data, task_id=load_into_json, execution_date=20220818T162653, start_date=20220818T163237, end_date=20220818T163240
[2022-08-18 16:32:40,266] {standard_task_runner.py:97} ERROR - Failed to execute job 69 for task load_into_json (Bash command failed. The command returned a non-zero exit code 1.; 1268)
[2022-08-18 16:32:40,313] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-08-18 16:32:40,684] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
